scripts/debug_ped_discrete.py

```
@@ -0,0 +1,75 @@
+
+
+def run():
+    env = make_env()
+
+    obs = env.reset()
+    ep_rewards = 0
+    for _ in range(10000):
+        if isinstance(obs, tuple):
+            action = select_action(obs[0])
+        else:
+            action = select_action(obs)
```

Ruff

Use ternary operator `action = select_action(obs[0]) if isinstance(obs, tuple) else select_action(obs)` instead of `if`-`else`-block

Replace `if`-`else`-block with `action = select_action(obs[0]) if isinstance(obs, tuple) else select_action(obs)`

(SIM108)